validation-test/stdlib/UnsafeBufferPointer.swift.gyb

```diff
@@ -746,6 +747,7 @@ UnsafeMutable${'Raw' if IsRaw else ''}BufferPointerTestSuite.test("subscript/${R
   defer { deallocateFor${Raw}Buffer(
       sliceMemory, count: replacementValues.count) }
 
+  // this produces a spurious compiler warning,, someone take a look lol?
```

Comment:
Minor point, but can we clean up this comment before merge?

Reply:
for some reason if the buffer on the line below is declared `var` the compiler complains that it’s never mutated, but if it’s declared `let` it doesn’t compile since a mutating member gets called on it.

Reply:
it’s out of the scope of this proposal but it’s something we should come back to later and sort out

Reply:
Sure; I was only talking about capitalization, punctuation, etc :)

Reply:
fixed